src/views/admin/SendMessage.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <div>
+        <div>
+            <h1 class="text-lg font-bold">Send Message</h1>
+            <div class="mt-2">
+                <textarea
+                rows="8"
+              cols="40"
+              type="text"
+              name="todo"
+              class="p-3 border"
+              placeholder="Enter Message ..."
+              v-validate="'required'" 
+              v-model="message"
+                           
+                >
+                </textarea>
+            </div>
+            <div class="border">
+                <div class="mb-3 text-2xl font-bold ml-1"><label class="text-base">Select Agents</label>
+                </div>
+                <select v-model="selectedUser" @change="addToArray()" :disabled="selectAll">
```

Comment:
add to Array???

Reply:
Yes! I need to have it all so that select all can work